app/models/schemas/doi/formats.json

```diff
@@ -0,0 +1,8 @@
+{
+  "title": "Formats",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "array",
+  "items": {
+    "$ref": "size.json"
```

Comment:
I don't think it makes a difference since sizes and formats are both just arrays of strings, but this points to size.json rather than format.json